model/src/main/proto/interaction_object.proto

```diff
@@ -57,27 +57,35 @@ message ListOfSetsOfHtmlStrings {
   StringList set_of_html_strings = 1;
 }
 
-// Structure for a ImageWithRegions object.
+// An image overlaid with labeled regions.
 message ImageWithRegions {
+  // A string that represents a filepath which will be prefixed with '[exploration_id]/assets.
   string image_path = 1;
-  repeated LabelRegions label_regions = 2;
-}
 
-message LabelRegions {
-  string label = 1;
-  Region region = 2;
-}
+  // A region of an image, including its shape and coordinates.
+  message LabelRegion {
+    // A name for a particular region.
+    string label = 1;
 
-message Region {
-  string region_type = 1;
-  NormalizedRectangle2D area = 2;
-}
+    message Region {
+      // A shape value for a particular region.
+      string region_type = 1;
+
+      message NormalizedRectangle2d {
+        message Point2d {
+          float x = 1;
+          float y = 2;
+        }
+        // Origin is top-left, increasing x is to the right, increasing y is down.
```

Comment:
Suggest adding documentation for ``NormalizedRectangle2d`` to describe what it is. This is also a nice note to move to it since it's more specifically for the context of the whole rectangle rather than this specific point.

Reply:
done